events/test/unit/.mbedignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Perhaps you could combine the different .mbedignore files.

Reply:
Done.